task_sdk/src/airflow/sdk/api/datamodels/_generated.py

```diff
@@ -1,7 +1,3 @@
-# generated by datamodel-codegen:
```

Comment:
Lets keep only the changes that you need for this file, yes the codegen makes some smaller changes that arent needed at times.

Reply:
updated, thanks!